src/shared/logger.ts

```diff
@@ -202,33 +210,35 @@ function makeDateString(type: 'filename' | 'logfile'): string {
     const isFilename: boolean = type === 'filename'
 
     return `${d.getFullYear()}${isFilename ? '' : '-'}` +
-    // String.prototype.padStart() is not available in Typescript...
-    `${padNumber(d.getMonth() + 1)}${isFilename ? '' : '-'}` +
-    `${padNumber(d.getDate())}${isFilename ? 'T' : ' '}` +
-    `${padNumber(d.getHours())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getMinutes())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getSeconds())}`
+        // String.prototype.padStart() is not in Nodejs version we're using...
```

Comment:
It's not related to TypeScript *or* NodeJS version. It's based on our build target (ES6) as defined in `tsconfig.json`. It is present in ES7+. Consider something like `// String.prototype.padStart() was introduced in ES7, but we target ES6.`

Reply:
Updated the existing comment again.

Regarding `toolkitTempFolder`. If I revert, I get errors stating dir exists when I re-run the Lambda. Will take work to address.

Reply:
✔️